lib/in/get-unused-packages.js

```diff
@@ -45,7 +49,11 @@ function checkUnused(currentState) {
         spinner.stop();
         const unusedDependencies = [].concat(depCheckResults.dependencies, depCheckResults.devDependencies);
         currentState.set('unusedDependencies', unusedDependencies);
-        currentState.set('missingFromPackageJson', depCheckResults.missing || {});
+
+        const cwdPackageJson = currentState.get('cwdPackageJson');
+        const missingFromPackageJson = _.omit(depCheckResults.missing || {},
+                    Object.keys(cwdPackageJson.dependencies), Object.keys(cwdPackageJson.devDependencies));
```

Comment:
I checked your code, you are not using the `withoutDev` in your `depCheckOptions`. You can use `result.missing` directly.